pynestml/utils/model_installer.py

```diff
@@ -61,23 +73,31 @@ def install_nest(target_path: str, nest_path: str) -> None:
     else:
         shell = False
 
+    # remove CMakeCache.txt if exists
+    cmakeCache = os.path.join(target_path, "CMakeCache.txt")
+    if os.path.exists(cmakeCache):
+        os.remove(cmakeCache)
```

Comment:
```suggestion
    cmake_cache = os.path.join(target_path, "CMakeCache.txt")
    if os.path.exists(cmake_cache):
        os.remove(cmake_cache)
```

Reply:
Please change the variable name in order to be consistent with the rest of the code.